strawberry/persistance/exceptions.py

```diff
@@ -0,0 +1,6 @@
+class PersistedQueryNotFoundError(Exception):
```

Comment:
Until now we have kept all the exceptions in `strawberry.exceptions (except for pydantic because it is still experimental). So I will move this exception with others.